backend/benefit/applications/api/v1/application_batch_views.py

```diff
@@ -190,7 +190,7 @@ def talpa_export_batch(self, request, *args, **kwargs) -> StreamingHttpResponse:
         )
 
         response = StreamingHttpResponse(
-            csv_service.get_csv_string_lines_generator(True),
+            csv_service.get_csv_string_lines_generator(True, True),
```

Comment:
I usually call functions with named arguments, at least with more than one argument. It makes the code clearer but also bulkier, so I guess it's a matter of preference.